### ISSUE TITLE
Reaper Scans: Changed their theme to a custom one

### ISSUE THREAD

**Issue by I-Am-C0der** [tachiyomiorg/tachiyomi-extensions#13747]
### Source information and language

Reaper Scans 1.3.31

### Steps to reproduce

1.Install Reaper Scans 1.3.31
2.Select Reaper Scans in Browse->Source

### Expected behavior

This should show list of Manhwa/Manhua available in the site

### Actual behavior

But it's showing **HTTP Error 405**

![Screenshot_2022-10-07-17-18-30-50_afd66b7916d19a60815f56273408b73e](https://user-images.githubusercontent.com/51527183/194546965-57b5eab9-fc97-4f1e-8309-2238376616fd.jpg)

### Tachiyomi version

0.13.6

### Android version

Android 11

### Other details

- The webview is working without any issues.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by TheKingTermux** [comment]
Duplicate of #13735

**Comment by alessandrojean** [comment]
That is another issue for the Brazilian Reaper Scans extension, so it's not a duplicate.

**Comment by RazredgeBR** [comment]
Why was #13639 closed? The app wasn't working then and isn't working yet, it wasn't even updated at all.

**Comment by suyash01** [comment]
multiple issues regarding reaper scans were closed due to not planned. don't know the exact reason.

**Comment by sahil-patel6** [comment]
Actually reaper scans is still in the midst of updating their site. They have yet to implement all features like search, bookmark, etc. So maybe the developer is waiting for that to finish. Atleast the search part is to be finished....

**Comment by tsunaproxD** [comment]
> Actually reaper scans is still in the midst of updating their site. They have yet to implement all features like search, bookmark, etc. So maybe the developer is waiting for that to finish. Atleast the search part is to be finished before the extension can be fixed and they have also changed the routes. So it will take time, it might take upto few weeks depending on the reaper scans team....

So this is the reason, thx that, need to wait and no more.

**Comment by sahil-patel6** [comment]
> > Actually reaper scans is still in the midst of updating their site. They have yet to implement all features like search, bookmark, etc. So maybe the developer is waiting for that to finish. Atleast the search part is to be finished before the extension can be fixed and they have also changed the routes. So it will take time, it might take upto few weeks depending on the reaper scans team....
> 
> So this is the reason, thx that, need to wait and no more.  

Yes

**Comment by h-hyuuga** [comment]
Closed by #13751

**Comment by sirzento** [comment]
After the latest extension upgrade (1.3.35) it is very buggy. The Reaper Scans extension isn't even listed anymore in the source section and on the migrating section there is just an id without icon where reaper scan was before.

**Comment by sahil-patel6** [comment]
> After the latest extension upgrade (1.3.35) it is very buggy. The Reaper Scans extension isn't even listed anymore in the source section and on the migrating section there is just an id without icon where reaper scan was before.

Yeah, facing the same issue. Temporarily i reverted back to previous version...

**Comment by suyash01** [comment]
I had to migrate all the comics back to reaper scans for it to work

**Comment by henrik9999** [comment]
The English **Reaper Scans** has been made into a standalone extension. To retain progress, install the new Reaper Scans extension available under the English section. If you haven't migrated already after the redesign, you will need to migrate from Reaper Scans to Reaper Scans to update the URLs. If you have already migrated, no extra migration is needed.

Note that **Reaper Scans**'s search is terrible so it may require a manual search or trying different search terms to find the correct entry to migrate to.

If you're running into issues, try restarting the app.

**Protip:** If you only use the English source, you can now uninstall the All/Multi Reaper Scans extension.

**Comment by I-Am-C0der** [comment]
**Issue Resolved.**
Steps to fix:
1.Uninstall Reaper Scans (All language version)
2.Install Reaper Scans 1.3.35 (English Version)
3.Migrate your Comics/Manhwas

Have fun...

**Comment by I-Am-C0der** [comment]
Issue Resolved.
Steps to fix:
1.Uninstall Reaper Scans (All language version)
2.Install Reaper Scans 1.3.35 (English Version)
3.Migrate your Comics/Manhwas

Have fun...

**Comment by tsunaproxD** [comment]
> 

This works perfectly.
